electron/main.js

```diff
@@ -211,8 +210,8 @@ const showMainWindow = () => {
       backgroundThrottling: false,
       nodeIntegration: false,
       preload: PRELOAD_JS,
-      webviewTag: true,
-    },
+      webviewTag: true
+    }
```

Comment:
We also decided to keep trailing commas ;)

Reply:
@ffflorian Absolutely right. But it is not in this projects eslint config. We have a discussion on how we are going to move forward with codestyle in general at https://github.com/wireapp/wire-webapp/pull/1729 that you are more than welcome to share your opinions in. For now I will go ahead and merge it in order to prepare our release.